### ISSUE TITLE
[Feature Request] ESP8266 compatible wrapper

### ISSUE THREAD

**Issue by nightgryphon** [greiman/SdFat-beta#83]
Please can you add an "official" wrapper to make this project compatible with ESP8266 libraries ( fs::File etc )?
It will be cool to keep using latest SdFat / SdFat-beta with regular ESP8266 libraries

**Comment by greiman** [comment]
Use this [ESP8266SdFat](https://github.com/earlephilhower/ESP8266SdFat/tree/46151c90a410a6f983f2f8c147e13086aaecdd8e).

**Comment by nightgryphon** [comment]
That project is used as part of an official Arduino ESP8266 library and based on SdFat 2.1.1. 
I've got an SD mount performance issue using that library. 
In my project SD card is shared between 2 MCUs: STM32 and ESP8266. At the moment any of two MCUs needs to use SD card it activates SPI pins and mount card than after file operation complete SPI is stopped and pins switched to input direction.
At the STM32 side everything work quite fast and fine while with ESP8266 mount operation takes about 5-7 _SECONDS_ which is awful slow and i can't figure out the reason for that.
I've checked and both MCUs access SD one at the time. There is no attempts to use card simultaneously. At any time one MCU attempts to mount card the other MCU pins are disabled. So the reason for such slow mount is not clear and i have no idea how to debug this with ESP8266
